File: c/1-sequence-struct/multiplication/main.c
// Write a program that asks for two integers and prints the multiplication.

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n1, n2, mul;

    printf("Enter the number to be multiplied: ");
    scanf("%d", &n1);
    printf("Enter the multiplier number: ");
    scanf("%d", &n2);

    mul = n1 * n2;
    printf("\nMultiplication of the entered numbers is %d.\n", mul);
    return 0;
}

File: java/1-sequence-struct/multiplication/src/Main.java
// Write a program that asks for two integers and prints the multiplication.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        // Instantiating object in java for data entry
        Scanner sc = new Scanner(System.in);

        int n1, n2, mul;

        System.out.print("Enter the number to be multiplied: ");
        n1 = sc.nextInt();
        System.out.print("Enter the multiplier number: ");
        n2 = sc.nextInt();

        mul = n1 * n2;
        System.out.println("\nMultiplication of the entered numbers is " + mul);


    }
}
